### PR TITLE
User and Restaurant Order Verification

### PR DESCRIPTION
**Pull request by rseln**
- Nuking order once confirm pickup is clicked
- Adding restaurant validation search thingy

### CODE REVIEW

**Comment by KarimAlatrash** [comment]
lgtm

**Review by KarimAlatrash — APPROVED** [review]

File: app/src/main/java/com/example/munchbox/ui/components/OrderSearchCard.kt [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+                Row(modifier = Modifier.weight(1f, false)) {
+                    TextField(
+                        modifier = Modifier.weight(1f).width(240.dp),
+                        label = { Text(text = "Order ID") },
+                        value = search.value,
+                        onValueChange = { search.value = it }
+                    )
+                    Spacer(modifier = Modifier.width(10.dp))
+                    ExtendedFloatingActionButton(
```
Comment (KarimAlatrash) [inline app/src/main/java/com/example/munchbox/ui/components/OrderSearchCard.kt]: disable if theres nothing in the text field

**Comment by KarimAlatrash** [comment]
other than the one comment it looks good to me

**Review by aaiss1 — APPROVED** [review]
lgtm!!!!!!!!!!

**Pull request merged**